src/blocks/form/captcha/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import { inputIcon as icon } from '../../../helpers/icons.js';
+import attributes from './attributes.js';
+import edit from './edit.js';
+import save from './save.js';
+import './editor.scss';
+
+registerBlockType( 'themeisle-blocks/form-captcha', {
```

Comment:
Do we need to make a new block? It will be harder for a user to discover this option that way. It should just be part of the parent block. Plus, this way a user can add multiple instances of recaptcha on one block (there's a way to avoid that tho) but even if we fix that, it will make the option hidden. What do you think?

Reply:
It can be only one instance via a toggle, but I made it a block like the others as default.

Reply:
Captcha isn't really a field that users will move around. If you don't know the captcha block exists, you will likely not notice it.